specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```diff
@@ -1368,6 +1372,79 @@
         }
       }
     },
+    "CacheDirectorySettings": {
+      "description": "Cache Directory Services settings.",
+      "properties": {
+        "usernameDownload" : {
+          "$ref": "#/definitions/CacheUsernameDownloadSettings",
```

Comment:
Are more Properties expected to be coming in the future outside of the username download settings?
Just asking in case the fields from username download settings could go directly into the Cache directory settings.

We have two conflicting examples above this, so it can go either way.
I see cache encryption settings which just goes on to reference another definition and has no other fields.
Then below that is cache security settings which directly has fields in it.

Reply:
yes. The SMB related settings will be another property of Cache Directory Services settings. I thought it'd be better to have additional definitions for those settings.